enhancements/proposals/1304-external-seeds/README.md

```diff
@@ -0,0 +1,156 @@
+	// See https://opensource.docs.scylladb.com/stable/kb/seed-nodes.html.
+	// For the first running node in a datacenter, the external seeds are the only seeds provided.
+	// For the remaining nodes, one other node from their datacenter is appended to the external seeds.
+	// +optional
+	ExternalSeeds []string `json:"externalSeeds,omitempty"`
+}
+```
+
+The provided `externalSeeds` are passed to the sidecar through a new `--external-seeds` flag. StatefulSet reconciliation in ScyllaDB operator is modified accordingly.
+The sidecar's strategy for choosing seeds to provide to ScyllaDB processes is modified to follow the below pseudocode:
+```go
+func chooseSeeds(externalSeeds []string, seedFromLocalDC string) []string {
+  if is first node in DC {
+    if externalSeeds not empty {
+      return externalSeeds
+    }
+    return seedFromLocalDC
+  }
+
+  return append(externalSeeds, seedFromLocalDC)
+}
+```
+
+It is important that the external seeds, and only the external seeds, are provided as seeds for the first running node in a datacenter, if they are at all provided by the end user. If external seeds were only appended to the node's own IP, it could result in creating split clusters in case of lack of connectivity between the datacenters.
+For the other nodes, a seed from their own DC is appended to the external seeds for increased resiliency.
+
+The above changes allow for configuring multi datacenter ScyllaDB clusters by manually providing external seeds.
+The below examples assume the two datacenters are to be deployed in one Kubernetes cluster.
+
+### Creating a multi datacenter ScyllaDB cluster
+To create a multi datacenter ScyllaDB cluster, an end user would first deploy a ScyllaCluster. For the purpose of this document, let's assume name `simple-cluster`, namespace `scylla1`, datacenter `us-east-1` and a single rack `us-east-1a`.
+To create the second datacenter, the end user would deploy a second ScyllaCluster with the same name, in a different namespace, with a distinct datacenter and rack names, and `.spec.externalSeeds` set to a list of IP address or domain names under which one or more nodes of the first datacenter are reachable from the second datacenter. 
+ScyllaDB operator will propagate the declared `externalSeeds` through a sidecar flag, and the sidecar will take it into account when configuring the seeds provided to ScyllaDB nodes of the second datacenter. The two datacenters will then form a single multi datacenter cluster.
+
+### Adding a new datacenter to an existing ScyllaDB cluster
+To add a new datacenter to an existing ScyllaCluster, the same procedure as above is followed.
+
+### Test Plan
+
+To test the proposed API extension, unit tests verifying that the external seeds are propagated to the sidecar will be added.
+Additionally, E2E tests should verify whether a single, multidatacenter ScyllaDB cluster is formed when additional datacenters with external seeds pointing to existing datacenter are created.
```

Comment:
should we test connecting 2 existing cluster as well?

Reply:
see the above comment